### ISSUE TITLE
Change project statuses

### ISSUE THREAD

**Issue by ellemindwave** [research-resource/research_resource#210]
Change 'Current' and 'Archived' project statuses in Redcap to:

-Recruiting participants
- Data analysis
- Writing up results
- Published
- Archived

**Comment by SimonLab** [comment]
duplicate of #33 ?

**Comment by ellemindwave** [comment]
@SimonLab Yes, this is a duplicate of #33 but the difference would be what is done in Redcap and then how this is reflected on the site.  If this is too much work now before user testing then that's okay - I have told the client that this would potentially be further down the line.
